src/Raven.Client/ServerWide/Operations/EditRevisionsForConflictsOperation.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class EditRevisionsForConflictsOperation : IServerOperation<EditRevisionsForConflictsOperationResult>
```

Comment:
Shouldn't be renamed to `ConfigureRevisionsForConflictsOperation`? For consistency with `ConfigureRevisionsOperation`

Reply:
done